DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h

```diff
@@ -11,75 +11,150 @@
  **/
 class CTPPSLocalTrackLite
 {
   public:
-    CTPPSLocalTrackLite() : rpId(0), x(0.), x_unc(-1.), y(0.), y_unc(-1.), time(0.), time_unc(-1.)
+
+    CTPPSLocalTrackLite() : rpId(0), x(0.), x_unc(-1.), y(0.), y_unc(-1.), tx(999.), tx_unc(-1.), ty(999.), ty_unc(-1.), 
+    chiSquaredOverNDF(-1.), pixelTrack_reco_info(ReconstructionInfo::invalid), numberOfPointsUsedForFit(0), time(0.), time_unc(-1.)
     {
     }
 
-    CTPPSLocalTrackLite(uint32_t pid, float px, float pxu, float py, float pyu, float pt=0., float ptu=-1.)
-      : rpId(pid), x(px), x_unc(pxu), y(py), y_unc(pyu), time(pt), time_unc(ptu)
-    {
+    CTPPSLocalTrackLite(uint32_t pid, float px, float pxu, float py, float pyu, float ptx, float ptxu, 
+        float pty, float ptyu, float pchiSquaredOverNDF, ReconstructionInfo ppixelTrack_reco_info, 
+        unsigned short pNumberOfPointsUsedForFit, float pt, float ptu)
+      : rpId(pid), x(px), x_unc(pxu), y(py), y_unc(pyu), tx(ptx), tx_unc(ptxu), ty(pty), ty_unc(ptyu), chiSquaredOverNDF(pchiSquaredOverNDF), 
+      pixelTrack_reco_info(ppixelTrack_reco_info), numberOfPointsUsedForFit(pNumberOfPointsUsedForFit), time(pt), time_unc(ptu)
+    { 
     }
 
     /// returns the RP id
-    uint32_t getRPId() const
+    inline uint32_t getRPId() const
     {
       return rpId;
     }
 
     /// returns the horizontal track position
-    float getX() const
+    inline float getX() const
     {
       return x;
     }
 
     /// returns the horizontal track position uncertainty
-    float getXUnc() const
+    inline float getXUnc() const
     {
       return x_unc;
     }
 
     /// returns the vertical track position
-    float getY() const
+    inline float getY() const
     {
       return y;
     }
 
     /// returns the vertical track position uncertainty
-    float getYUnc() const
+    inline float getYUnc() const
     {
       return y_unc;
     }
 
     /// returns the track time
-    float getTime() const
+    inline float getTime() const
     {
       return time;
     }
 
     /// returns the track time uncertainty
-    float getTimeUnc() const
+    inline float getTimeUnc() const
     {
       return time_unc;
     }
 
+    /// returns the track horizontal angle
+    inline float getTx() const
+    {
+        return tx;
+    }
+
+    /// returns the track horizontal angle uncertainty
+    inline float getTxUnc() const
+    {
+        return tx_unc;
+    }
+    
+    /// returns the track vertical angle
+    inline float getTy() const
+    {
+        return ty;
+    }
+    
+    /// returns the track vertical angle uncertainty
+    inline float getTyUnc() const
+    {
+        return ty_unc;
+    }
+    
+    /// returns the track fit chi Squared over NDF
+    inline float getChiSquaredOverNDF() const
+    {
+        return chiSquaredOverNDF;
+    }
+
+    /// returns the track reconstruction info byte
+    inline unsigned int getRecoInfo() const
+    {
+        return (unsigned int)pixelTrack_reco_info;
```

Comment:
why is the cast applied here (instead of e.g. returning ReconstructionInfo type)?
Note that the "full" LocalTrack type has ```ReconstructionInfo getRecoInfo() const```.